### PR TITLE
Feature/html structure

### PR DESCRIPTION
**Pull request by Andrey-1992**
## Subject:
Html structure elements
## Reason for Pull Request:
Upload initial HTML file
## Changes Implemented:
Build basic HTML Structure

### CODE REVIEW

**Comment by janitastic** [comment]
Really great start! Thank you for putting this together. You two rock!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     <section class='box1'>
+      <h2>New Activity</h2>
+      <section class='user-interaction'>
+         <div class='category-type'>
+           <h3>Select a category:</h3>
+           <button class='study-cat-btn'>Study</button>
+           <button class='meditate-cat-btn'>Meditate</button>
+           <button class='exercises-cat-btn'>Exercise</button>
+         </div>
+         <div class='user-spefications'>
```
Comment (LKessell) [inline index.html]: Minor typo, should be 'user-specifications' :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          </form>
+          <button class='user-specs-btn'>START ACTIVITY</button>
+         </div>
+      </section>
+     </section>
+     <section class='box2'>
+       <h2>Past Activities</h2>
+       <div class='past-act-msg'>
+         <p>You haven't logged any activities yet</p>
+         <p>Complete the form to the left to get started!</p>
```
Comment (janitastic) [inline index.html:39]: good catch on that p end tag!

**Review by LKessell — COMMENTED** [review]
Great structure, Andrey!

**Pull request merged**